pkg/lang/golang/queries/gocloud/file_bucket.scm

```diff
@@ -0,0 +1,18 @@
+(short_var_declaration
```

Comment:
no long var declarations? 😢

Reply:
i guess not, go is a simpleton

Reply:
This probably won't handle:
```go
var bucket, err = fileblob.OpenBucket(myDir, myConnector)
```
(presumably the (long) `var_declaration`)

or
```go
var bucket, err
bucket, err = fileblob.OpenBucket(myDir, myConnector)
```

Reply:
let me update the query, should be able to make it work without having to change any of the logic i believe